index.html

```diff
@@ -277,6 +278,7 @@
       "application/octet-stream",
       "application/x-sqlite3",
       "application/vnd.sqlite3",
+      "binary/octet-stream",
```

Comment:
being able to override this if you host you own instance doesnt seem like the worlds worst option either. If the PR were to pivot to that, I think it would make `datasette-lite` more pluggable